include/picongpu/particles/particleToGrid/derivedAttributes/WeightedVelocity.def

```diff
@@ -76,12 +77,8 @@ namespace picongpu
                     //! Get text name
                     HINLINE static std::string getName()
                     {
-                        if constexpr(T_direction == 0)
-                            return "particleVelocityX";
-                        else if constexpr(T_direction == 1)
-                            return "particleVelocityY";
-                        else
-                            return "particleVelocityZ";
+                        auto const componentNames = plugins::misc::getComponentNames(3);
+                        return "weightedVelocity/" + componentNames[T_direction];
```

Comment:
Try to avoid copy of temporary string?
```suggestion
                        auto componentNames = plugins::misc::getComponentNames(3);
                        return "weightedVelocity/" + std::move(componentNames[T_direction]);
```

Reply:
IMO without move it is easier to read. I checked it also in compiler explorer and the assembler without move looks better

https://godbolt.org/z/xYaavqeKs